backend/lib/endpoints/schema/organisations.js

```diff
@@ -40,6 +40,13 @@ const createOrganisationSchema = {
     .required(["location"]),
 };
 
+const createOrganisationAvatarSchema = {
+  params: strictSchema().prop("organisationId", S.string().required()),
+  body: strictSchema()
+    .prop("file", S.required())
+    .prop("userId", S.required()),
```

Comment:
```suggestion
  body: strictSchema().prop("file", S.required()),
```

The `userId` is attached to req via a decorator in the auth.js plugin so we don't need to include it in the body.

Reply:
I am working on the uploads ATM. Thanks.

Reply:
Ah good catch, I'll remove it from here.

Reply:
@joshmorel Resolved in this [commit](https://github.com/FightPandemics/FightPandemics/pull/1540/commits/b66fab6f892da514fd649f8cf9f5aaf27c382601).